### ISSUE TITLE
Ambiguous reexports on geobase 0.2.0 with constants 

### ISSUE THREAD

**Issue by navispatial** [navibyte/geospatial#122]
Version [0.2.0 of geobase](https://pub.dev/packages/geobase/versions/0.2.0) has some warnings when deployed on pub.dev:

```
exitCode: 255
stdout: [2022-06-17T19:18:06.182954] Starting...
Documenting geobase...
Initialized dartdoc with 183 libraries
Generating docs for library tiling from package:geobase/tiling.dart...
Generating docs for library meta from package:geobase/meta.dart...
Generating docs for library codes from package:geobase/codes.dart...
Generating docs for library coordinates from package:geobase/coordinates.dart...
Generating docs for library geobase from package:geobase/geobase.dart...
Generating docs for library projections_proj4d from package:geobase/projections_proj4d.dart...
Generating docs for library codes from package:geobase/constants.dart...
Generating docs for library vector from package:geobase/vector.dart...
Documented 8 public libraries in 34.4 seconds

stderr:   warning: ambiguous reexport of geodetic.minLongitude, canonicalization candidates: (geobase, codes, tiling) -> tiling (confidence 0.000)
    from geodetic.minLongitude: (file:///tmp/pub-dartlang-dartdocYSQQWY/pkg/lib/src/constants/geodetic.dart:10:7)
        geobase: 0.1777 - name is long (+0.01000), element location shares parts with name (+0.1667), element location parts start with parts of name (+0.001000)
        codes: 1.010 - embeds package name (+1.000), name is long (+0.01000)
        tiling: 1.010 - embeds package name (+1.000), name is long (+0.01000)
            Use {@canonicalFor geodetic.minLongitude} in the desired library's
            documentation to resolve the ambiguity and/or override dartdoc's
            decision, or structure your package so the reexport is less
            ambiguous.  The symbol will still be referenced in all candidates --
            this only controls the location where it will be written and which
            library will be displayed in navigation for the relevant pages. The
            flag --ambiguous-reexport-scorer-min-confidence allows you to set the
            threshold at which this warning will appear.
  warning: ambiguous reexport of geodetic.maxLatitudeWebMercator, canonicalization candidates: (geobase, codes, tiling) -> tiling (confidence 0.000)
    from geodetic.maxLatitudeWebMercator: (file:///tmp/pub-dartlang-dartdocYSQQWY/pkg/lib/src/constants/geodetic.dart:25:7)
        geobase: 0.1777 - name is long (+0.01000), element location shares parts with name (+0.1667), element location parts start with parts of name (+0.001000)
        codes: 1.010 - embeds package name (+1.000), name is long (+0.01000)
        tiling: 1.010 - embeds package name (+1.000), name is long (+0.01000)
  warning: ambiguous reexport of geodetic.minLatitude, canonicalization candidates: (geobase, codes, tiling) -> tiling (confidence 0.000)
    from geodetic.minLatitude: (file:///tmp/pub-dartlang-dartdocYSQQWY/pkg/lib/src/constants/geodetic.dart:16:7)
[...]
        geobase: 0.1777 - name is long (+0.01000), element location shares parts with name (+0.1667), element location parts start with parts of name (+0.001000)
        codes: 1.010 - embeds package name (+1.000), name is long (+0.01000)
        tiling: 1.010 - embeds package name (+1.000), name is long (+0.01000)
  error: failed to write file at: codes/codes-library.html
    for symbol codes: (file:///tmp/pub-dartlang-dartdocYSQQWY/pkg/lib/constants.dart:12:9)
    conflicting with file already generated by codes: (file:///tmp/pub-dartlang-dartdocYSQQWY/pkg/lib/codes.dart:12:9)
            Dartdoc generates a path and filename to write to for each symbol.
            codes conflicts with another symbol in the generated
            path, and therefore can not be written out.  Changing the name,
            library name, or class name (if appropriate) of one of the
            conflicting items can resolve the conflict.   Alternatively, use the
            @nodoc tag in one symbol's documentation comments to hide it.
Found 9 warnings and 1 error.
Unhandled exception:
encountered 1 errors
#0      Dartdoc.generateDocs (package:dartdoc/src/dartdoc.dart:231:9)
<asynchronous suspension>
#1      main (file:///project/pkg/pub_dartdoc/bin/pub_dartdoc.dart:32:19)
<asynchronous suspension>
```

At least `package:geobase/constants.dart` has some errors on it's definition dart file:

```dart
/// Geodetic and screen related constants.
///
/// This libary exports a subset of `package:geobase/geobase.dart`.
///
/// Usage: import `package:geobase/constants.dart`
library codes;

export 'src/constants/geodetic.dart';
export 'src/constants/screen_ppi.dart';
```

The library name should be `constants` not `codes`, of course.. Need to fix it..

**Comment by navispatial** [comment]
This was fixed [0.2.1-dev.0](https://pub.dev/packages/geobase/versions/0.2.1-dev.0/score).

**Comment by navispatial** [comment]
Published [0.2.1](https://pub.dev/packages/geobase/versions/0.2.1). Closing this ticket.
